mobile/reducers/tables-data/tables_data_reducer.js

```diff
@@ -14,7 +16,30 @@ export default function (state = initialTablesData, action) {
         ...state,
         inchargesList: action.payload || [],
       };
-
+    case CREATE_HEALTH_WORKER_REQUEST: {
+      const newChwList = _.clone(state.chwList);
+      const { newHealthWorker } = action.payload;
+      newHealthWorker.needSynchronize = true;
+      newChwList.push(newHealthWorker);
+      return {
+        ...state,
+        chwList: newChwList,
```

Comment:
you can use [ ...state.chwList, newHealthWorker ] instead of creating a copy of this array